bin/package.boilerplate.json

```diff
@@ -1,9 +1,10 @@
 {
   "devDependencies": {
-    "poetic": "^1.0.5"
+    "poetic": "^1.0.5",
+    "yarpm": "^0.2.1"
```

Comment:
Let's try to keep `poetic` as a single dependency. Also `yarpm` doesn't seem to be well maintained.

Reply:
Agreed, it would definitely be nicer to have poetic as a single dependency. :+1: